lib/modules/media/media_models.dart

```diff
@@ -380,10 +382,33 @@ class MediaInfo {
       }
     }
 
+    if (map['externalLinks'] != null) {
+      for (final link in map['externalLinks']) {
+        model.externalLinks.add(ExternalLink(link));
+      }
+    }
+
     return model;
   }
 }
 
+class ExternalLink {
+  final String url;
+  final String site;
+  final Color? color;
+
+  ExternalLink._({required this.url, required this.site, required this.color});
+
+  factory ExternalLink(Map<String, dynamic> map) => ExternalLink._(
+        url: map['url'],
+        site: map['site'],
+        color: map['color'] != null
+            ? Color(
+                int.parse(map['color'].substring(1, 7), radix: 16) + 0x8C000000)
+            : null,
```

Comment:
Not sure if you want to keep the colours. It looks good to me with a bit of transparency applied but it's up to you if you think it breaches too much on the design.

Reply:
They are a bit striking, but my main problem is that when using light mode, the dark text won't look good on the colored tiles... I propose we copy the AL site slightly 😉:
There's no need of any transparency and the scroll cards themselves won't be colored. Instead of passing just the text widget to the builder, wrap it in a row and to the left of it put a colored square. Make it rounded with radius `Consts.borderRadiusMin`. That should be enough. If I find it necessary, I may tweak it more later.

Reply:
Hey - This is looking a lot better imo. They're still a bit bright but it provides a nice accent. I could leave it here for now and you can tweak it later if you want?

![image](https://github.com/lotusgate/otraku/assets/48864420/5444017d-1682-4d22-9439-aec7dba59715)
![image](https://github.com/lotusgate/otraku/assets/48864420/cdf2206a-644c-4808-8be7-c2f4386907f0)